### PR TITLE
Adding return_url parameter to allow arbitrary return parameters on purchases

### PR DESCRIPTION
**Pull request by haraball**
LPC-615

### CODE REVIEW

File: laterpay/__init__.py [hunk 1]
```diff
@@ -305,6 +310,8 @@ def _get_web_url(self,
         params = self._sign_and_encode(data, base_url, method="GET")
         url = "{base_url}?{params}".format(base_url=base_url, params=params)
 
+        print(url)
```
Comment (doismellburning) [inline laterpay/__init__.py]: Please use `logging` rather than `print`

File: laterpay/__init__.py [hunk 2]
```diff
@@ -337,12 +346,15 @@ def get_add_url(self,
...
                     transaction_reference=None,
-                    consumable=False):
+                    consumable=False,
+                    return_url=None):
         """
         Get the URL at which a user can add an item to their invoice to pay later.
 
         https://www.laterpay.net/developers/docs/dialog-api#GET/dialog/add
         """
+        print(item_definition)
```
Comment (doismellburning) [inline laterpay/__init__.py]: De-`print` please

**Comment by doismellburning** [comment]
I know there's a dearth of documentation in here, and I think "fully documenting `__init__`" is definitely out of scope, but lets at least start documenting things as we add them!

**Comment by haraball** [comment]
FIAP for proper documentation on this?

**Comment by doismellburning** [comment]
#9 + #31 should cover this

**Comment by jdelic** [comment]
I also want to mention [LPC-539](https://laterpay.atlassian.net/browse/LPC-529) in this context. :cry:

**Comment by jdelic** [comment]
@doismellburning @haraball is this really still correctly tagged PFA?

**Comment by doismellburning** [comment]
@jdelic "PFA"?

**Comment by haraball** [comment]
@doismellburning pending (for) answer I assume

**Comment by haraball** [comment]
Let's add proper documentation in a separate PR and merge this?

**Comment by jdelic** [comment]
well, I meant "PA" then. :panda_face:

**Comment by doismellburning** [comment]
I realise it will stand amidst a sea of missing docs, and that we have issues for fixing that, but adding things without the faintest semblance of documentation is not the way to improve things

**Comment by jdelic** [comment]
@doismellburning @haraball did create laterpay/homepage#80 ?

**Comment by doismellburning** [comment]
Brilliant, so lets link to it

**Comment by haraball** [comment]
Boom!

**Pull request merged**